docs/zero_to_hero_guide/01_Local_Cloud_Inference101.ipynb

```diff
@@ -0,0 +1,257 @@
+    "local_client = LlamaStackClient(base_url=f'http://{HOST}:{LOCAL_PORT}')\n",
+    "cloud_client = LlamaStackClient(base_url=f'http://{HOST}:{CLOUD_PORT}')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "894689c1",
+   "metadata": {},
+   "source": [
+    "#### 2. Client Selection with Fallback\n",
+    "\n",
+    "The `select_client` function checks if the local client is available using a lightweight `/health` check. If the local client is unavailable, it automatically switches to the cloud client.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "ff0c8277",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "\u001b[33mUsing local client.\u001b[0m\n"
+     ]
+    }
+   ],
+   "source": [
+    "import httpx\n",
+    "from termcolor import cprint\n",
+    "\n",
+    "async def select_client(use_local: bool) -> LlamaStackClient:\n",
+    "    \"\"\"\n",
+    "    Selects the client based on the use_local flag.\n",
+    "\n",
+    "    Parameters:\n",
+    "    - use_local: bool, True to try the local client, False to use the cloud client.\n",
+    "\n",
+    "    Returns:\n",
+    "    - LlamaStackClient: the selected client instance.\n",
+    "    \"\"\"\n",
+    "    if use_local:\n",
+    "        try:\n",
+    "            async with httpx.AsyncClient() as http_client:\n",
+    "                response = await http_client.get(f'{local_client.base_url}/health')\n",
+    "                if response.status_code == 200:\n",
+    "                    cprint('Using local client.', 'yellow')\n",
+    "                    return local_client\n",
+    "        except httpx.RequestError:\n",
+    "            cprint('Failed to connect to local client.', 'red')\n",
+    "\n",
+    "    cprint('Using cloud client.', 'yellow')\n",
+    "    return cloud_client\n",
+    "\n",
+    "# Example usage: pass True for local, False for cloud\n",
+    "client = await select_client(use_local=True)\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9ccfe66f",
+   "metadata": {},
+   "source": [
+    "#### 3. Generate a Response\n",
+    "\n",
+    "After selecting the client, you can generate text using `chat_completion`. This example sends a sample prompt to the model and prints the response.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "5e19cc20",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from llama_stack_client.types import UserMessage\n",
+    "from termcolor import cprint\n",
+    "from llama_stack_client.lib.inference.event_logger import EventLogger\n",
+    "\n",
+    "async def get_llama_response(stream: bool = True, use_local: bool = True):\n",
+    "    client = await select_client(use_local)  # Selects the available client\n",
+    "    message = UserMessage(content='hello world, write me a 2 sentence poem about the moon', role='user')\n",
```

Comment:
as @subramen says, we should use 
```
message = {
  "role": "user",
  "content": "..."
}
```

here

Reply:
thanks, fixed this and the other occurrence on the other notebooks